### PR TITLE
Properly render static documents

### PR DESCRIPTION
**Pull request by heytitle**
This PR addresses the issue of returning wrong `content-type` for the `/docs/*` route, as mentioned in https://github.com/shogun-toolbox/shogun-web2/issues/42.

The change will modify the code for the following paths:
- http://shogun.ml (not expected any difference)
- http://shogun.ml/install (not expected any difference)
- http://shogun.ml/mission (not expected any difference)
- http://shogun.ml/docs/* (can be viewed from browsers)
    - http://shogun.ml/docs/ABOUT.md
    - http://shogun.ml/docs/DEVELOPING.md
    - http://shogun.ml/docs/EXAMPLES.md
    - http://shogun.ml/docs/INSTALL.md
    - http://shogun.ml/docs/INTERFACES.md
    - http://shogun.ml/docs/MISSION.md

### CODE REVIEW

File: shogun_web.py [hunk 1]
```diff
@@ -105,9 +105,9 @@ def sitemap():
 
 
 @app.route('/docs/<path:filename>')
-def docs_static(filename):
-    print app.root_path
-    return send_from_directory(DOCS_SUBMODULE_DIR, filename)
+def docs_static(filename, template='doc.html'):
+    doc_content = get_markuped(filename.lower(), "docs/%s" % filename)
```
Comment (vinx13) [inline shogun_web.py]: prefer the newer `.format`

Comment (heytitle) [reply shogun_web.py]: Do you mean something like this  `"docs/{}".format(filename)`?

Comment (vinx13) [reply shogun_web.py]: yes

Comment (heytitle) [reply shogun_web.py]: fixed at https://github.com/shogun-toolbox/shogun-web2/pull/43/commits/5d424a5d7589b768895d6c40395a85d2caad59e5

**Comment by karlnapf** [comment]
Thanks!

**Comment by karlnapf** [comment]
It is live

**Pull request merged**